examples/openapi3/jwt/app.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+'''
+Basic example of a resource server
+'''
+
+import time
+
+import connexion
+from werkzeug.exceptions import Unauthorized
+
+from jose import JWTError, jwt
+
+JWT_ISSUER = 'com.zalando.connexion'
+JWT_SECRET = '872f65270a71ca913766335276fb2c1b76b8ccdef749cdfc9a1598ac36d77c99'
+JWT_LIFETIME_SECONDS = 600
+JWT_ALGORITHM = 'HS256'
+
+
+def generate_token(user_id):
+    timestamp = _current_timestamp()
+    payload = {
+        "iss": JWT_ISSUER,
+        "iat": int(timestamp),
+        "exp": int(timestamp + JWT_LIFETIME_SECONDS),
+        "sub": str(user_id),
+    }
+
+    return jwt.encode(payload, JWT_SECRET, algorithm=JWT_ALGORITHM)
+
+
+def decode_token(token):
+    try:
+        return jwt.decode(token, JWT_SECRET, algorithms=[JWT_ALGORITHM])
+    except JWTError as e:
+        raise Unauthorized from e
```

Comment:
I don't think `raise ... from` is python 2.7 compatible.
You probably need to use `six.raise_from(...)`

Reply:
@dtkav It's not supposed to run on 2.7, please look at shebang.
BTW end of python 2.7 is coming :)

Reply:
It's not a matter of my personal preference (I'm also on python 3). Connexion explicitly supports python 2.7, so I think making the examples backwards compatible is the right thing to do.

If you don't want to do it, I'm happy to. I'd prefer to follow the rule of least surprise now, because otherwise I'll have to respond to a ticket about this example being "broken" later.

Reply:
I have made it python 2.7 compatible.

But it looks like [helloworld example](https://github.com/zalando/connexion/blob/dev-2.0/examples/openapi3/helloworld/hello.py) is also broken on 2.7